sdk/script/account-manager.ts

```diff
@@ -82,124 +95,121 @@ export class AccountManager {
     }
 
     public addAccessKey(machine: string, description?: string): Promise<AccessKey> {
-        return this.generateAccessKey()
-            .then((newAccessKey: string) => {
-                var accessKey: AccessKey = { name: newAccessKey, createdTime: new Date().getTime(), createdBy: machine, description: description };
-                return this.post(`/accessKeys/`, JSON.stringify(accessKey), /*expectResponseBody=*/ false)
-                    .then(() => accessKey);
-            });
+        var accessKeyRequest: AccessKey = { createdBy: machine, description: description };
+        return this.post(urlEncode `/accessKeys/`, JSON.stringify(accessKeyRequest), /*expectResponseBody=*/ true)
+            .then((response: JsonResponse) => response.body.accessKey);
     }
 
     public getAccessKey(accessKey: string): Promise<AccessKey> {
-        return this.get(`/accessKeys/${accessKey}`)
+        return this.get(urlEncode `/accessKeys/${accessKey}`)
```

Comment:
Should we just urlEncode inside the `.get(), .post()... ` etc methods? That way we don't have to attach the tag everywhere.